cobalt/android/apk/app/src/main/java/dev/cobalt/coat/CobaltActivity.java

```diff
@@ -350,6 +363,54 @@ protected void onCreate(Bundle savedInstanceState) {
     videoSurfaceView = new VideoSurfaceView(this);
     addContentView(
         videoSurfaceView, new LayoutParams(LayoutParams.MATCH_PARENT, LayoutParams.MATCH_PARENT));
+
+    initializeJavaBridge();
```

Comment:
If `initializeJavaBridge` needs a delay for initialize, maybe consider move it to `finishInitialization`.

Reply:
We need JavaBridge to be initialized ASAP, finishInitialization() runs after BrowserStartupController, it could be too late.